src/components/search/SearchForm.tsx

```diff
@@ -1,9 +1,15 @@
 import { SearchFormProps } from './type'
 
-function SearchForm({ getInput, keyword }: SearchFormProps) {
+function SearchForm({ getInput, keyword, changeIndexByKeyDown }: SearchFormProps) {
```

Comment:
![버그](https://github.com/wanted-pre-onboarding-team-12th-7/pre-onboarding-12th-3-7/assets/93248349/fc64f2fa-172f-4438-b3d8-3ade9f649f4f)

onBlur시에는 인덱스가 초기화 되고 있지만, suggestion이 변경 되었을때는 초기화 하는 로직이 없어, 
추천 검색어가 바뀌었을때 이전 인덱스가 유지되는 현상이 있습니다.

```suggestion
function SearchForm({ getInput, keyword, changeIndexByKeyDown }: SearchFormProps) {
  useEffect(() => {
      changeIndexByKeyDown.setIndex(0)
}, [suggestions])
```

임시로 useEffect를 사용하여 해결할 수 있을것 같은데, 더 좋은 방법이 있으면 좋겠네요 😓

Reply:
말씀해주신 useEffect로 훅 내에서 적용하니 문제 해결 되었습니다. 감사합니다!

Reply:
작동 확인 했습니다! 수고하셨어요 👍